rotkehlchen/api/rest.py

```diff
@@ -4018,6 +4018,14 @@ def get_user_added_assets(self, path: Optional[Path]) -> Response:
                 status_code=HTTPStatus.INSUFFICIENT_STORAGE,
             )
 
+        if path is None:
```

Comment:
Can you explain what this PR is doing, what was happening before and why you made this change?

Reply:
sry. was in discord but the PR should have contained it

Reply:
Updated the comment

Reply:
There is no comment about what you are doing here in the code though. When you or I read this 1 week later we are both going to wonder why it's doing this and what is it that it's doing.

You also need to edit the docs to specify all the different behaviours for the endpoint.

Also the related tests are failing.

Reply:
Improved. I've added one comment to the function explaining the behavior and fixed the tests. Also modified the docs to state clearly this behaviour. 

About the code we do the same at https://github.com/rotki/rotki/blob/develop/rotkehlchen/api/rest.py#L1680 and I didn't had problems to understand what it was doing but might be condition bias since I was working in a similar problem